docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-requests for your assets folder there.
-
-For example, if your project uses [express.js](https://expressjs.com/), below is
-a code sample you could add to your configuration:
-
-```JS
-app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
+```javascript
+import { initAll } from 'govuk-frontend'
+initAll()
 ```
-### Alternative solution
-
-Manually copy the images and fonts from `/node_modules/govuk-frontend/govuk/assets` into a public facing directory in your project. Ideally copying the files to your project should be an automated task or part of your build pipeline to ensure that the GOV.UK Frontend assets stay up-to-date.
-
-The default paths used for assets are `assets/images` and `assets/fonts`. **If your asset folders follow this structure, you will not need to complete the following steps.**
-
-To use different asset paths, also complete the below step(s).
-
-1. Set `$govuk-assets-path`, `$govuk-images-path` and `$govuk-fonts-path` in your project Sass file to point to the relevant directories in your project (this will override the defaults set in `/node_modules/govuk-frontend/govuk/settings/_assets.scss`). Make sure you do this in Sass before importing `govuk-frontend` into your project - see [Importing styles](#importing-styles).
-
-  Example 1:
-
-  ``` SCSS
-  // Include images from /application/assets/images and fonts from /application/assets/fonts
-  $govuk-assets-path: '/application/assets';
-
-  @import "govuk-frontend/govuk/all";
-  ```
-
-  Example 2:
 
-  ``` SCSS
-  // Include images from /images/govuk-frontend and fonts from /fonts
-  $govuk-images-path: "/images/govuk-frontend/";
-  $govuk-fonts-path: "/fonts/";
+If you're using a bundler that uses CommonJS like [Browserify](http://browserify.org/), you should use `require`:
 
-  @import "govuk-frontend/govuk/all";
-  ```
-
-2. Optional: You can also override the helpers used to generate the asset urls, for example if you are using sass-rails' asset-pipeline functionality. You can do this by setting `$govuk-image-url-function` to the name of the function(s) you wish to use. See `src/govuk/settings/_assets.scss` for more information and examples.
+```javascript
+const GOVUKFrontend = require('govuk-frontend')
+GOVUKFrontend.initAll()
+```
 
-## Include CSS and JavaScript
+### 5. Check it works
 
-Add the CSS and JavaScript code to your HTML template:
+Create an HTML file with the following code:
 
 ```html
 <!DOCTYPE html>
   <head>
-    <title>Example</title>
-    <link rel="stylesheet" href="assets/application.css">
+    <title>Example title</title>
+    <link rel="stylesheet" href="/{PATH-TO-CSS}/application.css">
   </head>
   <body>
-    <!-- Copy and paste component HTML-->
-    <button class="govuk-button">This is a button component</button>
-    <script src="assets/application.js"></script>
+    <button class="govuk-button">Example button</button>
+    <script src="/{PATH-TO-JAVASCRIPT}/all.js"></script>
```

Comment:
Should we consider using `main.js` or `application.js` instead of `all.js` so we avoid potential confusion with govuk-frontend/govuk/all.js and their application file

Reply:
The docs further up (and the current docs) recommend users copy or route to the `all.js` file, so I'm not sure changing the name would work? Unless we tweaked the docs further up to recommend changing the filename? Or add `{YOUR-PROJECT-PATH}` to the hrefs in the example? @36degrees - what do you think?